pom.xml

```
@@ -190,8 +196,8 @@
         <identity.outbound.auth.google.exp.version>${project.version}</identity.outbound.auth.google.exp.version>
         <carbon.identity.authenticator.oidc.imp.pkg.version.range>[5.0.0, 6.0.0)</carbon.identity.authenticator.oidc.imp.pkg.version.range>
         <!--Carbon Identity Framework Version-->
-        <carbon.identity.framework.version>5.5.0</carbon.identity.framework.version>
-        <carbon.identity.framework.package.import.version.range>[5.0.0, 6.0.0)</carbon.identity.framework.package.import.version.range>
+        <carbon.identity.framework.version>5.21.45</carbon.identity.framework.version>
+        <carbon.identity.framework.package.import.version.range>[5.14.67, 6.0.0]</carbon.identity.framework.package.import.version.range>
```

```suggestion
        <carbon.identity.framework.package.import.version.range>[5.14.67, 6.0.0)</carbon.identity.framework.package.import.version.range>
```

We usually keep the upper limit not included. (having `)` means version range up to 6.0.0 not 6.0.0. `]` means range with 6.0.0)

---

Fixed 389118ef7cce4b6917eb5b97117e2badacdcf060